### PR TITLE
Acceptance Test Added For Resend Invite

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to (link)?
- Acceptance test for #212 

#### Brief feature description
- Will fail until #212 merges.

### CODE REVIEW

File: spec/acceptance/user_edit_details_spec.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
 require 'acceptance_helper'
 require 'feature'
 require 'axe/rspec'
+require 'pry'
```
Comment (RamuKammagani-osi) [inline spec/acceptance/user_edit_details_spec.rb]: Remove debugger

File: spec/acceptance/user_edit_details_spec.rb [hunk 2]
```diff
@@ -107,15 +108,38 @@
...
 
     click_link 'User List'
     page_has_user_list_headers
   end
+
+  scenario 'user_details retain changes after clicking resend button' do
+    login
+    page_has_user_list_headers
+    sleep 2
+    # binding.pry
```
Comment (RamuKammagani-osi) [inline spec/acceptance/user_edit_details_spec.rb]: same here

File: spec/acceptance/support/user_list_page_helpers.rb [hunk 3]
```diff
@@ -51,6 +51,10 @@ def text_values(user_row)
     user_hash
   end
 
+  def any_user_link(value)
```
Comment (kelfink) [inline spec/acceptance/support/user_list_page_helpers.rb]: Good upgrade to the methods.  Let's rename the 'value' to something like 'row_number' for clarity.

Comment (kelfink) [reply spec/acceptance/support/user_list_page_helpers.rb]: Can you think of a better name than 'any_user_link' ? Maybe get_user_link ?

**Pull request merged**